conda-forge.yml

```diff
@@ -1,10 +1,12 @@
 bot:
   automerge: true
 build_platform:
+  linux_aarch64: linux_64
```

Comment:
Oh the rerender has turned on cross compiling for linux aarch64, rather than native?  @beckermr do you know anything about this?

Reply:
This test fails. I think it's a cross compiled target vs running the check on the host
`ldd $PREFIX/lib/libgalsim.so  # [linux]`

Reply:
Why is it running the tests if we have test on native only?

Reply:
Our migrators always move to cross compiling since it is more efficient for us. Feel free to move it back.